engine/src/main/java/com/datatorrent/stram/StreamingContainerAgent.java

```diff
@@ -271,10 +282,9 @@ public void addOperatorRequest(StramToNodeRequest r) {
 
         // On the input side there is a unlikely scenario of partitions even for inline stream that is being
         // handled. Always specifying a stream codec configuration in case that scenario happens.
-        InputPortMeta idInputPortMeta = getIdentifyingInputPortMeta(in);
-        StreamCodec<?> streamCodecInfo = getStreamCodec(idInputPortMeta);
-        Integer id = physicalPlan.getStreamCodecIdentifier(streamCodecInfo);
-        inputInfo.streamCodecs.put(id, streamCodecInfo);
+        final StreamCodec<?> streamCodec = getIdentifyingInputPortMeta(in).getStreamCodec();
```

Comment:
`idInputPortMeta` is not use other than to get streamCodec. It is not necessary to have it as a local variable.